astro/src/content/docs/get-started/core-concepts/integration-points.mdx

```diff
@@ -77,22 +78,28 @@ Recommended storage mechanisms include:
 
 * sending the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
 * storing it server side, in a session. The session is typically managed by a framework, and ideally adheres to the same cookie storage recommendations. Learn more about [server side sessions](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html). Please consult your framework documentation around securing sessions and data in sessions.
+* sending the token down to a native app and storing in a secure storage area such as iOS Keychain or Android Keystore.
 
-*Table 1. JWT Storage Options*
+Here is a table of characteristics of recommended JWT storage options.
 
-| Feature                                                                                                 | Cookie                                           | Server Side Session            |
-|---------------------------------------------------------------------------------------------------------|--------------------------------------------------|--------------------------------|
-| Allows for easy horizontal scaling                                                                      | Yes                                              | No                             |
-| Revocation possible                                                                                     | [Yes](/learn/expert-advice/tokens/revoking-jwts) | Yes                            |
-| Revocation easy                                                                                         | No                                               | Yes                            |
-| Sticky sessions or session datastore required                                                           | No                                               | Yes                            |
-| Token transparently sent on requests (to APIs, etc)                                                     | Yes                                              | N/A                            |
-| Token can be presented to APIs on domains other than that in which the cookie setting component resides | No                                               | Yes (via server side requests) |
+| Feature                                                                                                 | Cookie                                           | BFF/Server Side Session        | Native Secure Storage          |
```

Comment:
Should BFF be defined here? Maybe these are common knowledge, but I was not familiar with BFF, MTLS, or DPoP.

Reply:
I defined BFF above.